### ISSUE TITLE
[2019] Passage Re-ranking with BERT

### ISSUE THREAD

**Issue by rejasupotaro** [rejasupotaro/paper-reading#118]
## Summary

- They showed that a simple re-implementation of BERT for query-based passage re-ranking performs well on passage re-ranking tasks.

## Link

https://arxiv.org/pdf/1901.04085.pdf

## Notes

They used a pre-trained BERT model and fine-tune it for their re-ranking tasks with this cross-entropy loss:

<img width="240" alt="Screenshot 2020-01-09 at 10 18 07" src="https://user-images.githubusercontent.com/883148/72059130-5854f700-32c9-11ea-9554-7f3bfa72ecbd.png">

## BibTeX

```
@article{DBLP:journals/corr/abs-1901-04085,
  author    = {Rodrigo Nogueira and
               Kyunghyun Cho},
  title     = {Passage Re-ranking with {BERT}},
  journal   = {CoRR},
  volume    = {abs/1901.04085},
  year      = {2019},
  url       = {http://arxiv.org/abs/1901.04085},
  archivePrefix = {arXiv},
  eprint    = {1901.04085},
  timestamp = {Fri, 01 Feb 2019 13:39:59 +0100},
  biburl    = {https://dblp.org/rec/bib/journals/corr/abs-1901-04085},
  bibsource = {dblp computer science bibliography, https://dblp.org}
}
```
